video/agon.h

```diff
@@ -395,6 +396,7 @@
 #define TESTFLAG_AFFINE_TRANSFORM	1	// Affine transform test flag
 
 #define FEATUREFLAG_FULL_DUPLEX	0x0101	// Full duplex UART comms flag
+#define FEATUREFLAG_TILE_ENGINE	0x0300	// Tile engine flag (layers commands)

```

Comment:
the choice of flag number here is _slightly_ arbitrary, as there is not yet a standard for test/feature flag numbering - mostly because we have to few currently defined

here is a very rough proposal for a numbering standard:

| high-byte prefix | meaning |
| -- | -- |
| 00 | Buffered command extension settings / experimental test flags |
| 01 | Communications system settings/configuration |
| 02 | VDU system settings and configuration (see #265 ) |
| 03 | Graphics system extension settings (such as tile engine/layers) |

there's a draft PR #267 that currently uses 02xx numbered flags for some VDP protocol settings - those will be renumbered to use 01xx numbering.